dist/meteo-france-weather-card.js

```diff
@@ -330,19 +333,17 @@ class WeatherCard extends LitElement {
           ? html`
             <span class="pluie">
               ${
-                  rainForecastObj.state != "unknown"
-                  ? html`
+                  html`
                       ${
                         getRainForecast(rainForecastObj).map(
                           forecast => html`
-                            <span class="pluie-element" style="background-color: ${forecast[1]}" title="${forecast[2] + " " + ((forecast[0] == 0) ? "actuellement" : "dans " + forecast[0] + " min")}">
+                            <span class="pluie-element" style="opacity: ${forecast[1]}" title="${forecast[2] + " " + ((forecast[0] == 0) ? "actuellement" : "dans " + forecast[0] + " min")}">
```

Comment:
```suggestion
                            <span class="pluie-element" style="opacity: ${forecast[1]}" title="${forecast[2] + " " + (forecast[0] == 0 ? "actuellement" : "dans " + forecast[0] + " min")}">
```

---

OR for better readability:
```suggestion
                            <span class="pluie-element"
                                  style="opacity: ${forecast[1]}"
                                  title="${forecast[2] + " " + (forecast[0] == 0 ? "actuellement" : "dans " + forecast[0] + " min")}">
```